src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -317,3 +317,10 @@ def test_passlib(pyi_builder):
     pyi_builder.test_source("""
         import passlib.apache
         """)
+
+
+@importorskip('publicsuffix2')
+def test_publicsuffix2(pyi_builder):
+    pyi_builder.test_source("""
+        import publicsuffix2
```

Comment:
This doesn't appear to be enough. I can compile and run this without the hook and it still works. Try:
```suggestion
        import publicsuffix2
        publicsuffix2.PublicSuffixList()
```
That gives me the error without the hook.

Reply:
Good point, thanks. How do you go on to test this? I naively expected the pyi_builder thing to throw an error if execution succeeds in absence of a hook.